supervision/dataset/formats/coco.py

```diff
@@ -159,16 +173,29 @@ def detections_to_coco_annotations(
     return coco_annotations, annotation_id
 
 
+def get_coco_class_index_mapping(annotations_path: str) -> Dict[int, int]:
+    coco_data = read_json_file(annotations_path)
+    classes = coco_categories_to_classes(coco_categories=coco_data["categories"])
+    class_mapping = build_coco_class_index_mapping(
+        coco_categories=coco_data["categories"], target_classes=classes
+    )
+    inv_class_mapping = {v: k for k, v in class_mapping.items()}
+    return inv_class_mapping
+
+
 def load_coco_annotations(
     images_directory_path: str,
     annotations_path: str,
     force_masks: bool = False,
+    use_iscrowd: bool = True,
```

Comment:
I'ts a bit weird that by default `use_iscrowd=True` in `load_coco_annotations` but `use_iscrowd=False` in `coco_annotations_to_detections`. Let's unify that.

Reply:
Agree! :+1: 
:heavy_check_mark: Unified to `use_iscrowd=True`